package.json

```diff
@@ -91,5 +90,6 @@
       "lint:pre-commit"
     ],
     "silent": false
-  }
+  },
+  "browserslist": ["last 2 versions"]
```

Comment:
Netlify was throwing a ton of warnings that were overloading and causing the build to fail by having this in the postcss config. Works the same as having a browserslist - this is now the accepted way of doing it.